openreview/conference/invitation.py

```diff
@@ -302,6 +302,9 @@ def __init__(self, conference, reveal_authors, reveal_submission, hide_fields):
                     'values-regex': '[^;,\\n]+(,[^,\\n]+)*',
                     'required': False,
                     'order': 2
+                },
+                'keywords': {
```

Comment:
no need to make this change, I already added it in hide_fields.

Reply:
@purujitgoyal can you remove this change? 

the variable `hide_fields` should have the key `keywords`. Not all the invitations have this field.

Reply:
Done.